Basic-Recruit-Training/README.adoc

```diff
@@ -1,8 +1,86 @@
-Basic-Recruit-Training
------------------------
+= Pointers
+
+One of the most important concepts in C/C++ programming is pointers.
+A pointer is a very similar to other primitive data types such as int.
+Instead of containing a value a pointer will instead contain an address.
+This address is the location in memory of another variable.
+To initialize a pointer use an asterisk after the type that the pointer will be pointing to.
+
+For example:
+
+----
+int * p;
+----
+
+Will create a variable p, which is a pointer that will be able to point to a variable of type int.
+In order to assign our pointer p to the address of a variable we will need to use the address operator, &.
+Take note of the following example:
+
+----
+int x;
+int *p;
+
+x = 3;
+p = &x;
+----
+
+Printing the value of p will print the address of x.
+This will look something like 0x7ffc9b657c7c.
+The 0x is notation to show that the following number is in hexadecimal.
+This means that the value for x is stored on your machine at the memory location 0x7ffc9b657c7c.
+The real power of pointers comes from the use of the dereferencing operator, which is also an asterisk.
+Lets look back at the code example.
+
```

Comment:
In this example, I think it would be useful to illustrate the difference between * during declaration "int *p" and dereferencing operator when assigning a variable